src/error.rs

```diff
@@ -85,12 +85,21 @@ pub enum Error {
     #[error("Invalid quantity [{value}]")]
     InvalidQuantity { value: String },
 
+    #[error("Invalid quantity [{value}]")]
+    InvalidCpuQuantity { value: String },
+
     #[error("Invalid quantity unit [{value}]")]
     InvalidQuantityUnit { value: String },
 
     #[error("No quantity unit provided for [{value}]")]
     NoQuantityUnit { value: String },
 
+    #[error("Unsupported Precision [{value}]. CPU quantities can only be given without a unit or with 'm")]
```

Comment:
Not sure if we can copy this but thats whats written in the kubernetes docs.
```suggestion
    #[error("Unsupported Precision [{value}]. Kubernetes doesn't allow you to specify CPU resources with a precision finer than 1m. Because of this, it's useful to specify CPU units less than 1.0 or 1000m using the milliCPU form; for example, 5m rather than 0.005")]
```